nss-crystal-cobras/BangazonAPI
# Jd employee t6

# Description

Adding my Employee controller and the models for computer, employee, computerEmployee, and department.

## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made
git fetch all--
git checkout "BranchYouAreReviewing"

You can test the code by running tests in postman on the employees URL

Get individual, Get all, and post should be working fine.

http://localhost:5000/api/employees/


# Checklist:
